src/Card/Card.js

```diff
@@ -37,11 +38,11 @@ export class Card extends Component{
             <h2 className="movie-release">{moment(release_date).format('LL')}</h2>
             
             {this.props.favorites && this.props.status &&
-            <button className="favorite-movie" onClick={(event) => this.toggleFavorite(event)}>❤️</button>
+            <button className="favorite-movie" onClick={(event) => this.toggleFavorite(event)}><span role="img" aria-label="heart-filled">❤️</span></button>
             }
```

Comment:
when we get to the refactoring part of this project we should import a react-icon library so we can make use of better iconography